corebehrt/modules/cohort_handling/outcomes.py

```
@@ -64,18 +104,118 @@ def match_concepts(
+
+    @staticmethod
+    def _check_combination_args(args: dict):
+        if PRIMARY not in args:
+            raise ValueError(
+                f"{PRIMARY} must be defined in the combinations dictionary"
+            )
+        if SECONDARY not in args:
+            raise ValueError(
+                f"{SECONDARY} must be defined in the combinations dictionary"
+            )
+        timestamp_source = args.get(TIMESTAMP_SOURCE, None)
```

Ruff (0.8.2)

Use `args.get(TIMESTAMP_SOURCE)` instead of `args.get(TIMESTAMP_SOURCE, None)`

Replace `args.get(TIMESTAMP_SOURCE, None)` with `args.get(TIMESTAMP_SOURCE)`

(SIM910)